gdal/doc/source/programs/gdal2xyz.rst

```diff
@@ -0,0 +1,97 @@
+.. _gdal2xyz:
+
+================================================================================
+gdal2xyz
+================================================================================
+
+.. only:: html
+
+    Translates a raster file into xyz format.
+
+.. Index:: gdal2xyz
+
+Synopsis
+--------
+
+.. code-block::
+
+    gdal2xyz [-help]
+        [-skip factor]
+        [-srcwin xoff yoff xsize ysize]
+        [-b band]* [-allbands]
+        [-csv]
+        [-srcnodata value] [-dstnodata value]
+        src_dataset [dst_dataset]
+
+Description
+-----------
+
+The :program:`gdal2xyz` utility can be used to translates a raster file into xyz format.
+gdal2xyz is an alternative to `gdal_translate` with `-of=xyz` with other options,
```

Comment:
backticks around `gdal2xyz`

"with other" -> "with other options" ?

Reply:
I've rephrased that.